golden_response/staging/get_chart_data.json

```diff
@@ -223,10 +223,42 @@
       "2020-07-01": 4572,
       "2020-07-02": 4750,
       "2020-07-03": 4849,
-      "2020-07-04": 5077
+      "2020-07-04": 5077,
+      "2020-07-05": 5273,
+      "2020-07-06": 5408,
+      "2020-07-07": 5478,
+      "2020-07-08": 5552,
+      "2020-07-09": 5678,
+      "2020-07-10": 5863,
+      "2020-07-11": 5983,
+      "2020-07-12": 6298,
+      "2020-07-13": 6542,
+      "2020-07-14": 6725,
+      "2020-07-15": 6951,
+      "2020-07-16": 7046,
+      "2020-07-17": 7131,
+      "2020-07-18": 7300,
+      "2020-07-19": 7456,
+      "2020-07-20": 7795,
+      "2020-07-21": 8046,
+      "2020-07-22": 8321,
+      "2020-07-23": 8533,
+      "2020-07-24": 8719,
+      "2020-07-25": 8833,
+      "2020-07-26": 9041,
+      "2020-07-27": 9215,
+      "2020-07-28": 9359,
+      "2020-07-29": 9612,
+      "2020-07-30": 9731,
+      "2020-07-31": 9913,
+      "2020-08-01": 10321,
+      "2020-08-02": 10626,
+      "2020-08-03": 10794,
+      "2020-08-04": 11030
     },
     "place_name": "Santa Clara County",
-    "place_dcid": "geoId/06085"
+    "place_dcid": "geoId/06085",
+    "provenance_domain": "nytimes.com"
```

Comment:
@pradh this might be one example that we want to update golden data when there is new data (field). So for each cache, need to determine on what field should be considered as "redundant" or not for truth assertion